lib/rules/function-calc-no-invalid/index.js

```diff
@@ -156,7 +155,8 @@ function getNumber(mathExpression) {
 			? getNumber(right)
 			: null;
 
-	if (_.isNil(leftValue) || _.isNil(rightValue)) {
+	// eslint-disable-next-line eqeqeq
```

Comment:
The `smart` option of the `eqeqeq` rule can be helpful.
See https://eslint.org/docs/rules/eqeqeq#smart

```js
/*eslint eqeqeq: ["error", "smart"]*/
```

Reply:
Yeah, but I'd rather not touch the ESLint config here since there's an upstream eslint-config-stylelint.

Reply:
Also, do note we have at least one more case like this, so it would indeed make sense to switch to `smart` upstream

Reply:
I opened the issue: https://github.com/stylelint/eslint-config-stylelint/issues/69